src/Evolution/Systems/CurvedScalarWave/Equations.hpp

```diff
@@ -92,4 +86,203 @@ struct ComputeDuDt {
       const Scalar<DataVector>& gamma1,
       const Scalar<DataVector>& gamma2) noexcept;
 };
+
+/*!
+ * \brief Compute fluxes for the scalar-wave system in curved spacetime.
+ *
+ * \details The expressions for fluxes is obtained from \cite Holst2004wt by
+ * taking the principal part of equations 15, 23, and 24, and replacing
+ * derivatives \f$ \partial_k \f$ with the unit normal \f$ n_k \f$ (c.f. Gauss'
+ * theorem). This gives:
+ *
+ * \f{align*}
+ * F(\psi) &= -(1 + \gamma_1) N^k n_k \psi \\
```

Comment:
Please use our standard notation, such as alpha for lapse, etc.

Reply:
changed them everywhere